### PR TITLE
Update swipe actions colors and logic

### PR DESCRIPTION
**Pull request by LunarX**

### CODE REVIEW

File: app/src/main/res/layout/swipe_illustration_content.xml [hunk 1]
```diff
@@ -47,14 +47,17 @@
 
     <com.google.android.material.imageview.ShapeableImageView
         android:id="@+id/description1"
-        android:layout_width="236dp"
+        android:layout_width="0dp"
         android:layout_height="7dp"
         android:layout_marginStart="@dimen/marginStandard"
         android:layout_marginTop="12dp"
         android:layout_marginBottom="@dimen/marginStandardSmall"
         android:background="#C4C4C4"
```
Comment (KevinBoulongne) [inline app/src/main/res/layout/swipe_illustration_content.xml:55]: Hardcoded color in a layout, is it on purpose?

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/swipe_illustration_content.xml (hunk 1 above)
Comment (LunarX) [reply app/src/main/res/layout/swipe_illustration_content.xml:55]: Yes, I recreated the illustration to have created control on how it's displayed across multiple width of screen. But these color don't really represent anything else in the app

**Review by JorisBodin — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/ic_bin.xml [hunk 2]
```diff
@@ -18,6 +18,7 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
     android:width="24dp"
     android:height="24dp"
+    android:tint="@color/swipeActionIconColor"
```
[the remark this answers is not in the document]
Comment (LunarX) [reply app/src/main/res/drawable/ic_bin.xml:21]: So would I, but the DragDropSwipeRecyclerView library did not account for tinting its icons. The custom recyclerview only accepts a drawable res id with no other way of tinting

[the remark this answers is not in the document]
Comment (JorisBodin) [reply app/src/main/res/drawable/ic_bin.xml:21]: Will have to propose an RP to this lib

**Review by JorisBodin — APPROVED** [review]

**Pull request merged**